src/ImageRunModal.scss

```diff
@@ -0,0 +1,18 @@
+@import '@patternfly/patternfly/layouts/Grid/grid.css';
+
+// Limit the max-width more the override in podman.scss which is 40%
+#run-image-dialog-memory, #run-image-cpu-priority {
+     max-width: 15%;
+}
+
+#memory-unit-select {
+     max-width: 20%;
+}
+
+#run-image-dialog-memory-limit-checkbox, #run-image-dialog-cpu-priority-checkbox {
```

Comment:
...Something like `.dialog-run-checkbox` and apply the class to both elements as a `className` in the JSX would be better if we do have to special case this at the checkbox level.